lib/mocha/any_instance_method.rb

```diff
@@ -14,7 +14,7 @@ def original_method_body
     end
 
     def store_original_method
-      @original_method = original_method_owner.instance_method(method_name)
+      @original_method = stubbee.instance_method(method_name)
```

Comment:
Can you explain how this is definitely equivalent? i.e. how you know there is no change in behaviour?

Reply:
This is definitely equivalent because I've essentially inlined the `original_method_owner` call here, i.e. `original_method_owner` returns `stubbee`.